src/Readme.md

```diff
@@ -1,2 +1,27 @@
-# Readme.md
-Azure/CIQS-Azure-CliExtension
\ No newline at end of file
+# Description
+This repository contains the implementation of CIQS(Cloud Intelligence Quickstart) Azure Cli extension. It help AI solution authoring and provisioning platform (a.k.a CIQS) reach out to a broader set of customers, including data scientists, data engineers, developers and ISVs.
+
+# Build Instructions

```

Comment:
State system pre-requisites here. For example, git clone repo, install python 3.x, install az cli, etc. It will work if you reflink to some instruction. Let's assume that the audience of this tutorial knows nothing about azure cli and nothing about CIQS; this tutorial is written to walk them through the journey.

Reply:
Okay